src/style.css

```diff
@@ -1 +1,64 @@
-/* Styles go here */
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  background: #e5e5e5;
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  background: #fff;
+  box-shadow: 2px 4px rgba(0, 0, 0, 0.05);
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo.image {
+  display: block;
+  width: 40px;
+}
+
+.nav__list {
+  display: flex;
+  justify-content: center;
+  list-style: none;
+  text-align: center;
+  margin: 0;
+}
+
+.nav__item:not(:first-child) {
+  margin-left: 20px;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  font-weight: 500;
+  font-size: 12px;
+  align-items: center;
+  color: #000;
+  text-transform: uppercase;
+  text-decoration: none;
+  text-align: center;
+  padding: 23px 0;
+}
+
+.nav__link:hover {
+  color: #00acdc;
```

Comment:
Move all repeatable colors to the CSS Variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)